scylla/Cargo.toml

```diff
@@ -38,7 +38,7 @@ thiserror = "1.0"
 itertools = "0.11.0"
 bigdecimal = "0.2.0"
 num-bigint = "0.3"
-tracing = "0.1.36"
+tracing = { version = "0.1.36", features = ["log"] }
```

Comment:
I think it is wrong for us to unconditionally enable this feature. Our crate does not need it at all in order to function properly. The decision whether to enable the feature or not should be on the application developers (as it controls global behavior) - they can explicitly depend on `tracing` in their application and enable `log` feature.

Reply:
I'm not very familiar with how Cargo handles dependencies, resolves versions etc. I know it's possible to have 2 copies (2 versions) of the same dependency. What is user supposed to do if they want to enable `log` feature in `tracing` used by our driver and not include another `tracing` copy?
Is it possible for us to have a feature that enables `log` feature in `tracing`? I feel like this would be a bit easier to use.

Reply:
> I'm not very familiar with how Cargo handles dependencies, resolves versions etc. I know it's possible to have 2 copies (2 versions) of the same dependency. What is user supposed to do if they want to enable `log` feature in `tracing` used by our driver and not include another `tracing` copy?

They need to specify a version constraint such that the cargo resolver will choose the same version. I don't know all the intricacies of the algorithm either, but I'm pretty sure that if you specify:

```toml
[dependencies]
tracing = { version = "0.1", features = ["log"] }
```

then, given that we have `tracing = "0.1.36"` in our depencencies, the resolver will choose the same version and instance of the `tracing` crate. Users can verify if they succeeded by looking at `Cargo.lock`.

> Is it possible for us to have a feature that enables `log` feature in `tracing`? I feel like this would be a bit easier to use.

Yes, it is possible. However, I still think it's weird for us to add such a feature. If we were to add it, going further with that logic, all other library crates that use `tracing` internally should have such a feature as well - this does not scale. Users can easily enable the necessary feature in `tracing` for their application  with the method I outlined.